src/Language/Haskell/Liquid/GHC/Plugin.hs

```diff
@@ -70,7 +73,11 @@ import           Language.Haskell.Liquid.Bare
 import           Language.Haskell.Liquid.UX.CmdLine
 
 import           Optics
+import           Data.Bitraversable                       (bitraverse)
 
+data LiquidCheckException =
+  KillPluginWithSuccess
```

Comment:
This is a fantastic place to explain the meaning of `LiquidCheckException`, i.e. how it differs from other errors, and explain when `KillPluginWithSuccess` is delivered.

Reply:
Changed.